public/app/features/scopes/selector/types.ts

```diff
@@ -14,3 +15,6 @@ export interface TreeNode {
   query: string;
   children?: Record<string, TreeNode>;
 }
+export interface RecentScope extends Scope {
+  parentNode?: ScopeNode;
```

Comment:
As we only set this in the first scope, as all the scopes in the same "selection" will have the same parent, maybe we can change this to be something like `recentScopes = Array<{parent?: ScopeNode, scopes: Scope[]}>`? I think that more represents the intent here.